hack/build-go.sh

```diff
@@ -14,7 +14,8 @@ os::log::install_errexit
 
 # only works on Linux for now, all other platforms must build binaries themselves
 if [[ -z "$@" ]]; then
-  if os::build::detect_local_release_tars $(os::build::host_platform_friendly) >/dev/null; then
+  if [[ "${OS_RELEASE:-}" != "n" ]] && \
```

Comment:
I'm reusing the semantics of OS_RELEASE from hack/build-images.sh since it seemed to me that the use case was similar.  Would you prefer that a different mechanism be used?